concourse/scripts/spec/configure_aruba.rb

```diff
@@ -0,0 +1,37 @@
+require "aruba/rspec"
+
+RSpec.configure do |rspec_config|
+  rspec_config.before(:all) do
+    # We run scripts with git commands, and as we don't want to mess around
+    # with our own repository, we setup the workdir in a different location.
+    @workdir = Dir.mktmpdir("paas-cf-tmpdir-")
+    # Aruba requires the workdir path to be a relative directory
+    relative_workdir = Pathname.new(@workdir).relative_path_from(Pathname.pwd)
+
+    Aruba.configure do |aruba_config|
+      aruba_config.working_directory = relative_workdir
+      aruba_config.home_directory = @workdir
+      aruba_config.command_search_paths = [
+        File.join(File.join(__FILE__, "..", "..")),
+      ]
+      if ENV["DEBUG"]
```

Comment:
You might have already discussed this in meatspace and maybe it's already been fixed, but it'd be good to include any error messages that you experienced when this happened.